### DISCUSSION TITLE
[5.0] Menu page header and Article header style

### DISCUSSION THREAD

**Discussion by 1inkin** [joomla/joomla-cms#47054] [2. Cold Features]
### Steps to reproduce the issue
1. Create Menu Item with Type = Single Article
2. In tab Page Display for this Menu Item select Show Page Heading = Show and Page Heading = for example "Menu page header"
3. Open this menu page in front

### Expected result
Menu page header and Article header have different styles

### Actual result
Menu page header and Article header have the same style
div class="page-header"

### Additional comments
In Joomla 4 Article header had the property "itemprop="headline"" which allowed you to set a different style, but in Joomla 5 this property is gone.

**Comment by drmenzelit** [comment]
In Joomla 5 the code for schema.org (e.g. "itemprop="headline") in the views were removed, since we now have a schema.org plugin.
If you display the page heading and the title of the article the `div` elements have the same class, but there are different headings: h1 and h2, so you can style them differently. 
![grafik](https://github.com/joomla/joomla-cms/assets/9153168/665dd552-6d1d-4ec1-ad02-64810f80063f)

You can also use something like that:
`.com-content-article > .page-header:first-of-type {
  background-color: gold;
}
`

**Comment by 1inkin** [comment]
This option only works if there is a menu header and a article header. But if you open a article without a menu header with this style, then the header of the article will receive this style. but this style should only be applied to menu header. I defined three different header combinations: 1. article header only. 2. menu header and article header, when the article is single article of this menu. 3. menu header and article header when the article is in the category of this menu. Without a separate class, it is impossible to set styles so that headers are displayed correctly in all cases.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42739">issues.joomla.org/tracker/joomla-cms/42739</a>.</sub>

**Comment by chmst** [comment]
please test #44505
